parachain/runtime/snowbridge/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "snowbridge-runtime"
-description = "Artemis Snowbridge Runtime"
+description = "Snowbridge Snowbridge Runtime"
```

Comment:
Dupe

Reply:
Might be correct though? Can we rename the runtime/testnet?

Reply:
I still need to rename the runtime pallets appropriately, so will fix the dupe along with that change.